### PR TITLE
feat(meta-seo): add meta seo tags

### PR DESCRIPTION
**Pull request by hellodit**
Hii there. 
I just add meta SEO for home page and post page.

### CODE REVIEW

File: exampleSite/config.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ params:
   # Enable float footnotes.
   # Default to true
   enableFootnotes: true
+  sitename: "Hugo Tania is Amazing"
```
Comment (WingLim) [inline exampleSite/config.yaml:14]: I prefer to use Camel-Case.
In this case, it will be `siteName`.

Comment (hellodit) [reply exampleSite/config.yaml:14]: I've already updated to Camel-Case

**Comment by WingLim** [comment]
Notify me, when you finished it.

**Comment by hellodit** [comment]
Already finish, or you have any idea add more meta for SEO?

**Comment by WingLim** [comment]
> Already finish, or you have any idea add more meta for SEO?

It's good for me now.

**Review by WingLim — APPROVED** [review]

**Pull request merged**